exoatlas/__init__.py

```diff
@@ -2,5 +2,6 @@
 from .visualizations import *
 from .models import *
 from .telescopes import *
+from .observing import *
```

Comment:
@MiriCaradonna , just to show what's happening here, anything that gets put into `exoatlas/__init__.py` will be available when we say `from exoatlas import *`. I added an `observing/` directory to the code directory, with your `transits.py` inside of it.